azurerm/internal/services/managementgroup/management_group_resource.go

```diff
@@ -220,8 +220,7 @@ func resourceArmManagementGroupRead(d *schema.ResourceData, meta interface{}) er
 		return err
 	}
 
-	recurse := true
-	resp, err := client.Get(ctx, id.Name, "children", &recurse, "", managementGroupCacheControl)
+	resp, err := client.Get(ctx, id.Name, "children", utils.Bool(true), "", managementGroupCacheControl)
```

Comment:
`recurse` wants to be split out so it's clearer what this is:

```suggestion
	recurse := utils.Bool(true)
	resp, err := client.Get(ctx, id.Name, "children", recurse, "", managementGroupCacheControl)
```